### PR TITLE
run the formatter over source files

### PR DESCRIPTION
**Pull request by devoncarew**
- run the formatter over the  source files
- tweak the .gitignore file

### CODE REVIEW

**Comment by devoncarew** [comment]
@DanTup, looks like this is failing:

- https://travis-ci.org/Dart-Code/Dart-Code/jobs/156691867#L99
- https://github.com/Dart-Code/Dart-Code/blob/master/src/providers/dart_completion_item_provider.ts#L89

**Comment by devoncarew** [comment]
Possibly a VSCode insiders API?

**Comment by DanTup** [comment]
Urrrgggg...

That thing did change in Insiders, but to keep my new branch clean, I made the change back in master because I thought it was using it as an interface not a class ;-(

I'll change it to call the constructor (I did initially, but this was a smaller change) today. If you need to build in the meantime, feel free to just delete those two lines (`additionalTextEdits` and `command`).

File: src/editors.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import * as vs from 'vscode';
 
 export function hasActiveDartEditor(): boolean {
   if (!vs.window.activeTextEditor)
-  	return false;
```
Comment (DanTup) [inline src/editors.ts]: This file seems to have two-space indenting?

**Comment by DanTup** [comment]
Merged this since the failure isn't related; though worth fixing the 2-space file. Maybe setting `editor.detectIndentation: false` in this project would fix it from sticking with spaces where they already exist?

I really wish Code would get format-on-save already! ;(

**Comment by DanTup** [comment]
I've removed the bad code online, so failure should at least be fixed!

**Pull request merged**